src/client/tile.cpp

```diff
@@ -1834,15 +1865,12 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 			to produce a valid string.
 		*/
 		else if (str_starts_with(part_of_name, "[png:")) {
-			Strfnd sf(part_of_name);
-			sf.next(":");
 			std::string png;
 			{
-				std::string blob = sf.next("");
+				std::string blob = part_of_name.substr(5);
```

Comment:
BTW we can make this use a string view with a trivial modification to our base64 util.

Reply:
I'm aware. There's a bunch more code that would immediately benefit from string_view.
I might have time to prepare a PR in a 1-2 weeks, but feel free to also do it.